maquetado/scss/titles.scss

```diff
@@ -0,0 +1,24 @@
+@import 'colors';
+@import 'variables';
+
+.title {
+  align-items: center;
+  color: $black;
+  display: flex;
+  font-size: $text-big;
+  padding-left: 51px;
+  position: relative;
+  height: $title-decoration-height;
+
+  &::before {
+    background: (url(assets/triangle.svg));
```

Comment:
It's not necessary wrapper `(url(assets/triangle.svg))` you just put `url(assets/triangle.svg)` and below you have the property `background-repeat` so I consider that you should change the currently property `background` for `background-image`